### ISSUE TITLE
Emoji on TextPath rendered incorrectly when parent Text is rotated

### ISSUE THREAD

**Issue by lachtos** [react-native-community/react-native-svg#841]
There is an issue when rendering emoji on a TextPath while the parent Text of the TextPath is rotated. The emoji is rendered out of place. 

`#textPath` is just a straight line

```
<Text alignmentBaseline="middle" textAnchor="start" rotation={135}>
  <TextPath href="#textPath">
    <TSpan>Text with emoji 😃</TSpan>
  </TextPath>
</Text>
```

![image](https://user-images.githubusercontent.com/16613421/48366532-e417c700-e6b6-11e8-91d6-54b073b7531d.png)

**Comment by msand** [comment]
@lachtos Ah, thanks for reporting this. Forgot to test that while implementing it.
If you have time to look at this, then these are the relevant parts of the code:

Android: 26 lines of code https://github.com/react-native-community/react-native-svg/blob/38310677e3daafd57db622d236fea296d5c8f930/android/src/main/java/com/horcrux/svg/TSpanView.java#L862-L868

iOS: 33 lines of code https://github.com/react-native-community/react-native-svg/blob/72a53f373028e4c1d89286c1e6c93c7a229ebc90/ios/Text/RNSVGTSpan.m#L818-L841

**Comment by msand** [comment]
@lachtos Could you provide a full replication? The example you used for the screenshot looks quite useful for debugging/development. Btw, is it an issue on only iOS or Android, or on both? I notice the iOS code doesn't take rotation into account, while it seems Android does. But haven't made any test/example for this yet.

**Comment by msand** [comment]
@lachtos This seems to be fixed in https://github.com/react-native-community/react-native-svg/commit/abb17bc7f0985c0b5eaa85ead7351e5c74a731f8

**Comment by lachtos** [comment]
@msand Sorry I didn't reply to your previous comment, notification got lost in the emails. Will absolutely check the fix out later.

**Comment by msand** [comment]
Published v9.2.4 with the fixes.

**Comment by lachtos** [comment]
Thanks!
However, seems to be working incorrectly on an Android 5.1 device, looks the same as in the picture above.
